src/main/java/Step2/StringAdder.java

```diff
@@ -0,0 +1,57 @@
+package Step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAdder {
+    public int sum(String input) {
+        String delim = "";
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(input);
+        if (m.find()) {
+            delim = m.group(1);
+            input = m.group(2);
+        }
+
+        int result = 0;
+        String[] splits = input.split("");
+
+        for (String s : splits) {
+            result = update_result(s, delim, result);
+        }
+
+        return result;
+    }
+
+    private int update_result(String word, String delim, int result){
```

Comment:
```suggestion
    private int updateResult(String word, String delim, int result){
```
자바의 네이밍 컨벤션은 카멜 케이스입니다